### PR TITLE
Unified environment variable parsing and allow arguments from CLI

### PR DESCRIPTION
**Pull request by nlordell**
In order to unify how parameters to the driver are processed, we now have a single location where all the parameters are specified. This also allows for parameters to be passed in by command line instead of strictly using environment variables.

### Test Plan

CI.

### CODE REVIEW

**Review by e00E — APPROVED** [review]
Unifying environment variables and command line arguments is a great idea.

**Review by fleupold — APPROVED** [review]
Nice fix of the week 🚀 

<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia2.giphy.com%2Fmedia%2F3ohzdIuqJoo8QdKlnW%2Fgiphy-downsized-medium.gif&token=sweet" data-gitmeme-token="sweet"></a>

File: driver/src/main.rs [hunk 1]
```diff
@@ -13,51 +13,91 @@ mod util;
...
+use url::Url;
+
+#[derive(Debug, StructOpt)]
+#[structopt(
+    name = "driver",
+    about = "Gnosis Exchange protocol driver.",
+    rename_all = "kebab"
+)]
+struct Options {
+    /// The log fiter to use.
```
Comment (josojo) [inline driver/src/main.rs]: typo: fiter

**Review by josojo — APPROVED** [review]
Love it

**Pull request merged**